lib/ecl/ecl_coarse_cell.cpp

```diff
@@ -287,6 +287,17 @@ const int * ecl_coarse_cell_get_box_ptr( const ecl_coarse_cell_type * coarse_cel
 
 
 /*****************************************************************/
+void ecl_coarse_cell_reset_active_index(ecl_coarse_cell_type * coarse_cell, int active_value) {
+    if (active_value & CELL_ACTIVE_MATRIX)
```

Comment:
I presume the `active_value` will be used to determine whether this `coarse_cell` has to be updated during an update. Here, we can probably disregard that value an directly set both the active indices to -1

Reply:
I'm removing the active_value parameter and just setting them unconditionally inside the function.